src/app/modules/map/components/map/map.component.ts

```diff
@@ -559,11 +560,13 @@ export class MapComponent implements OnInit, OnDestroy, AfterViewInit {
   }
 
   private updateMapView() {
-    this.mapService.updateMapView({
-      bounds: this.map.getBounds(),
-      center: this.map.getCenter(),
-      zoom: this.map.getZoom(),
-    });
+    if (this.updateMapViewOnExtentChange && this.isActive.value) {
```

Comment:
Å sjekke `this.isActive.value` her tror jeg er det som faktisk løser oppgaven. Hvorfor updateMapView kalles i det hele tatt når man er i listevisninga kunne man sjekket ut hvis man skulle kommet enda lengre til bunns i problemene her.

Men uansett: Det å sjekke `this.isActive.value` her gjør at man ikke får noen observasjoner i kartet når man starter appen første gangen. Uten at jeg har sjekket det veldig nøye tror jeg det er fordi `updateMapView` kalles her når kartet lastes første gangen:

https://github.com/NVE/regObs4/blob/a0eee2514fafb9ae4c2e9987650d31b8271d9827/src/app/modules/map/components/map/map.component.ts#L297-L306